mail_alias_manager/db/models/receiving_alias.py

```diff
@@ -0,0 +1,14 @@
+from sqlalchemy.sql.schema import Column
+from sqlalchemy_utils.types.email import EmailType
+
+from ..db import DB, MODEL
+
+
+class ReceivingAlias(MODEL):
```

Comment:
Maybe call this `RecipientAlias`

```suggestion
class RecipientAlias(MODEL):
```

Reply:
I think ReceivingAlias is better

Reply:
The idea was that we reduce the amount of different words used here. `RecipientAlias` is comprised of the same words used in the attributes.